modules/retrieving-the-logs-with-the-oc-or-kubectl-command.adoc

```diff
@@ -13,12 +13,7 @@ You can use either the `oc` or `kubectl` command to obtain logs from your runnin
 +
 [source,terminal]
 ----
-$ oc auth can-i get pods <1>
-----
-+
-[source,terminal]
-----
-$ oc auth can-i get /logs/ <1>
+$ oc auth can-i get pods && oc auth can-i get /logs/ <1>
```

Comment:
@Stringy Do we need "/" after "/logs"?

Reply:
Yes I believe we do - I found this command as an example in the kubectl reference. There's an alternative approach `<kubectl or oc> auth can-i get pods --subresource=logs` which may be better ?